### ISSUE TITLE
No temporal listing in powersearch "content:..."

### ISSUE THREAD

**Issue by monnypython** [eikek/docspell#980]
When I do a full text search in the PowerSearch with "**content**:[String]", all documents are listed under "RESULTS" instead of chronologically structured.

![grafik](https://user-images.githubusercontent.com/42573701/127919929-60e87093-d4cd-42b9-a71c-50ef6811e876.png)

**Comment by eikek** [comment]
This is intended. When using only fulltext search, the ordering is used as returned from solr. Because solr returns the items ordered by relevance wrt search term. If the query is extended with another search condition (which requires to ask the database, too), it will be sorted by date (and will be slower).

**Comment by monnypython** [comment]
Ah ok. I didn't realize that. This time I have probably not read the document carefully enough: D.

Your explanation makes sense. You can close the ticket.

**Comment by eikek** [comment]
don't worry at all about this! it wasn't easy for me to find it.. and it is confusing to see a change in the ordering. not sure, maybe it's time for small one-time info bubbles in the app

**Comment by monnypython** [comment]
Yes. Sometimes it would be nice to have a bit more Info in place.
